### ISSUE TITLE
Plugin is die

### ISSUE THREAD

**Issue by vietnux** [floatinghotpot/cordova-plugin-facebookads#109]
Plugin is die width FacebookSDK 4.++
FacebookSDK 4.26 or 4.28 is not support

The build app is immediately closed

**Comment by kyawswarthwin** [comment]
same problem here

**Comment by hodaifa82** [comment]
did you solve this issue I'm having same problem is there any solution?

**Comment by kyawswarthwin** [comment]
You can install from this github url. I already solved it.
ionic cordova plugin add https://github.com/kyawswarthwin/cordova-plugin-facebookads

**Comment by vietnux** [comment]
@kyawswarthwin your url too having same problem :|

**Comment by kyawswarthwin** [comment]
@vietnux remove old plugin first.

**Comment by Defcon0** [comment]
@kyawswarthwin If it's working, maybe you could write a pull request for @floatinghotpot ?

**Comment by hodaifa82** [comment]
@@kyawswarthwin I'm using phonegapbuild  did it works also for it ?? if yes what Cordova version you use

**Comment by hodaifa82** [comment]
@@kyawswarthwin I'm using phonegapbuild  did it works also for it ?? if yes what Cordova version you use

**Comment by vietnux** [comment]
@kyawswarthwin no. It `s new version of android sdk, method is change should plugin error.

**Comment by eyalish** [comment]
> 
> 
> @@kyawswarthwin I'm using phonegapbuild did it works also for it ?? if yes what Cordova version you use

Where can you find this plugin for PhoneGap builder? is it available?

**Comment by max-in-bc** [comment]
Is anyone able to get this plugin (or facebook audience network ads, in general) working on android with cordova/phonegap anymore? It has stopped working for me for at least a few weeks now

**Comment by vietnux** [comment]
> 
> 
> Is anyone able to get this plugin (or facebook audience network ads, in general) working on android with cordova/phonegap anymore? It has stopped working for me for at least a few weeks now

All in one :D

**Comment by halkoy** [comment]
> You can install from this github url. I already solved it.
> ionic cordova plugin add https://github.com/kyawswarthwin/cordova-plugin-facebookads

hii  @kyawswarthwin ,
your plugin facebook sdk is old, please upgrade new sdk for android. Facebook return that sdk is old. we can not show ads. please upgrade

**Comment by merbin2012** [comment]
Try this https://github.com/merbin2012/cordova-plugin-codeplay-facebookads-free
